File: udemy/pascal/projetos/ExeAvaliacao.pas
program ExeAvaliacao;
var
   prova1, prova2, prova3, media : real;
begin
     writeln('informe a nota da primeira prova: ');
     read(prova1);
     writeln('informe a nota da segunda prova: ');
     read(prova2);
     writeln('informe a nota da terceira prova: ');
     read(prova3);
     media := (prova1*2 + prova2*3 + prova3*5)/10;
     write('a media das provas eh de: ', media);
     readln();
end.

File: udemy/java/Exeavaliacao/src/Exeavaliacao/ExeAvaliacao.java
package Exeavaliacao;
import javax.swing.JOptionPane;
public class ExeAvaliacao {

	public static void main(String[] args) {
		float prova1, prova2, prova3, media;
		prova1= Float.parseFloat(JOptionPane.showInputDialog("informe a nota da primeira prova: "));
		prova2 = Float.parseFloat(JOptionPane.showInputDialog("informe a nota da segunda prova: "));
		prova3 = Float.parseFloat(JOptionPane.showInputDialog("informe a nota da segunda prova: "));
		prova1 *= 2;
		prova2 *=3;
		prova3 *= 5;
		media = (prova1+prova2+prova3)/10;
		JOptionPane.showMessageDialog(null,"A media das provas foi de: "+media);
		
		
	}

}
